docs/appendices/backend-writers-guide.rst

```diff
@@ -771,71 +771,6 @@ other update/remove functionality at a later stage.
   must be added to the zone. ``rrset`` can be empty in which case the
   method is used to remove a RRset.
 
-DNS update support
-------------------
-
-To make your backend DNS update compatible, it needs to implement a
```

Comment:
Perhaps we should keep this paragraph and remove the stuff below indeed

Reply:
The dns update section is listed twice in this file see line 708-772